include/adaflo/navier_stokes.h

```diff
@@ -183,6 +183,12 @@ class NavierStokes : public FlowBaseAlgorithm<dim>
     return navier_stokes_matrix;
   }
 
+  const LinearAlgebra::distributed::BlockVector<double> &
+  get_rhs() const
```

Comment:
I thought I suggested to rename the function to get `get_system_rhs()`, since there are other rhs variables in this class..

Reply:
I also thought a bit about the naming. In the end, I decided to stick in line with https://github.com/kronbichler/adaflo/blob/79d43cad7f49c1683bda8a7fc62bace5dddf3f25/include/adaflo/navier_stokes.h#L180-L184. However, I agree that since there are multiple rhs, it might be better to be more specific.